tests/padders.py

```diff
@@ -109,3 +112,27 @@ def pad_test_kwargs(self, unchanged_slices, **input_param):
                             inv = padder.inverse(result)
                             assert_allclose(im, inv, type_test=False)
                             self.assertEqual(inv.applied_operations, [])
+
+    def pad_test_pending_ops(self, input_param, input_shape):
```

Comment:
would be great to have a test about lazily combining multiple crop/pad operations chained in different ordering will give the expected consistent outcomes.

Reply:
Hi @wyli, since we don't have the compose logic in this PR, do you mean combining multiple operations manually?

Reply:
Yes

Reply:
Hi @wyli, I have added a test about lazily combining multiple pad operations chained in different ordering in this commit. https://github.com/Project-MONAI/MONAI/pull/6031/commits/81de9e2f1b6aa82ad3ff3cb9754609a36a22c537
But here I have one question, I only test multiply pad operations with the same pad mode. If with different pad modes, which mode should I specify to `apply_transforms`?

Reply:
yes, in theory when we lazily evaluate the pending operations we should check the compatibility of the padding modes across the transforms https://github.com/Project-MONAI/MONAI/blob/b9e17e8186cb25c8781e2c82de3b4915081df48b/monai/transforms/lazy/functional.py#L83-L85 the `is_compatible_apply_kwargs` API is defined, for now we haven't implemented any specific rules. In the first version, I don't thinks it's a high-priority feature..